deps/internal/client/client.go

```diff
@@ -26,8 +28,21 @@ const (
 var (
 	protocol = or(os.Getenv(VariableProtocol), DefaultProtocol)
 	baseURL  = or(os.Getenv(VariableBaseURL), DefaultBaseURL)
+	systemOS = runtime.GOOS
+	arch     = runtime.GOARCH
```

Comment:
I'd just reference these inline. No need to create intermediary variables for them.

Reply:
Changed